.eslintrc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "@alvarosabu/eslint-config-vue",
```

Comment:
@lisilinhart do we have an internal `eslint` config for formatting inside Storyblok?